.ci/ci_job_flags.sh

```diff
@@ -75,6 +75,9 @@ init_ci_flags() {
 	export METRICS_JOB_BASELINE=""
 	# Configure test to use Kata SHIM V2
 	export SHIMV2_TEST="true"
+        # Install skopeo in the guest OS (for Confidential Containers)
+        # Values: "no|yes"
+        export SKOPEO="no"
```

Comment:
This is going to conflict with https://github.com/kata-containers/tests/pull/5065